recipe/CMakeLists.txt

```
@@ -0,0 +1,134 @@
+    "${CMAKE_CURRENT_LIST_DIR}/src/cmsplugin.c"
+    "${CMAKE_CURRENT_LIST_DIR}/src/cmsps2.c"
+    "${CMAKE_CURRENT_LIST_DIR}/src/cmssamp.c"
+    "${CMAKE_CURRENT_LIST_DIR}/src/cmssm.c"
+)
+
+if(BUILD_SHARED_LIBS)
+    target_compile_options(lcms PRIVATE -DCMS_DLL_BUILD)
+    target_compile_options(lcms PUBLIC -DCMS_DLL)
+endif()
+
+target_compile_options(lcms PRIVATE -DUNICODE -D_UNICODE)
+target_include_directories(lcms PRIVATE "${CMAKE_CURRENT_LIST_DIR}/include")
+set_target_properties(lcms PROPERTIES PUBLIC_HEADER "${CMAKE_CURRENT_LIST_DIR}/include/lcms2.h;${CMAKE_CURRENT_LIST_DIR}/include/lcms2_plugin.h")
+
+add_executable(jpegicc
+    "${CMAKE_CURRENT_LIST_DIR}/utils/jpgicc/iccjpeg.c"
+    "${CMAKE_CURRENT_LIST_DIR}/utils/jpgicc/iccjpeg.h"
+    "${CMAKE_CURRENT_LIST_DIR}/utils/jpgicc/jpgicc.c"
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common/vprf.c"
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common/xgetopt.c"
+)
+find_package (JPEG REQUIRED)
+if (JPEG_FOUND)
+  include_directories(${JPEG_INCLUDE_DIRS})
+  target_link_libraries (jpegicc ${JPEG_LIBRARIES} -static)
+endif (JPEG_FOUND)
+target_link_libraries (jpegicc lcms -static)
+target_include_directories(jpegicc PRIVATE 
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common"
+    "${CMAKE_CURRENT_LIST_DIR}/include"
+)
+
+add_executable(tifficc
+    "${CMAKE_CURRENT_LIST_DIR}/utils/tificc/tificc.c"
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common/vprf.c"
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common/xgetopt.c"
+)
+find_package (TIFF REQUIRED)
+if (TIFF_FOUND)
+  include_directories(${TIFF_INCLUDE_DIRS})
+  target_link_libraries (tifficc ${TIFF_LIBRARIES} -static)
+endif (TIFF_FOUND)
+target_link_libraries (tifficc lcms -static)
+target_include_directories(tifficc PRIVATE 
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common"
+    "${CMAKE_CURRENT_LIST_DIR}/include"
+)
+
+add_executable(transicc
+    "${CMAKE_CURRENT_LIST_DIR}/utils/transicc/transicc.c"
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common/vprf.c"
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common/xgetopt.c"
+)
+target_link_libraries (transicc lcms)
+target_include_directories(transicc PRIVATE 
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common"
+    "${CMAKE_CURRENT_LIST_DIR}/include"
+)
+
+add_executable(linkicc
+    "${CMAKE_CURRENT_LIST_DIR}/utils/linkicc/linkicc.c"
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common/vprf.c"
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common/xgetopt.c"
+)
+target_link_libraries (linkicc lcms)
+target_include_directories(linkicc PRIVATE 
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common"
+    "${CMAKE_CURRENT_LIST_DIR}/include"
+)
+
+add_executable(psicc
+    "${CMAKE_CURRENT_LIST_DIR}/utils/psicc/psicc.c"
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common/vprf.c"
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common/xgetopt.c"
+)
+target_link_libraries (psicc lcms)
+target_include_directories(psicc PRIVATE 
+    "${CMAKE_CURRENT_LIST_DIR}/utils/common"
+    "${CMAKE_CURRENT_LIST_DIR}/include"
+)
+
+install(TARGETS lcms jpegicc tifficc transicc linkicc psicc
```

```suggestion
install(TARGETS lcms jpgicc tificc transicc linkicc psicc
```
I don't think it is wise to change the name of the executables